release_notes/43-release-notes.adoc

```diff
@@ -254,6 +255,13 @@ Vulnerability deferral management for host (`/node`) and platform (`/cluster`) v
 * Fixed an issue with editing user-defined vulnerability reports in version 4.3 that were created in a previous version and linked to a specific report scope. When editing the report in version 4.3, the report scope reference was missing, and the system returned an error message.
 * Updated and removed golang dependencies to address reported vulnerabilities, including false positives.
 
+[id="resolved-in-version-432_{context}"]
+=== Resolved in version 4.3.2
+
+*Release date*: 14 December 2023
+
+Fixed an issue with manual delegated scanning that caused Central to crash.
```

Comment:
I can see that the wording here is consistent with the rest of the wording on this page, so I won't suggest a revision, but if this page is ever significantly updated, it probably would be best to use complete sentences (i.e., "This release fixed and issue...") instead of fragments.